src/swarm-util/Swarm/Language/Syntax/Direction.hs

```diff
@@ -126,7 +126,7 @@ instance ToJSON Direction where
 -- | Direction name is generated from the deepest nested data constructor
 -- e.g. 'DLeft' becomes "left"
 directionSyntax :: Direction -> Text
-directionSyntax d = from $ directionJsonModifier $ case d of
+directionSyntax d = T.pack $ directionJsonModifier $ case d of
```

Comment:
I don't like using `from` either, but how about `into @Text`?

Reply:
Well, `T.pack` is shorter than that :smile: